docs/src/content/docs/zh-cn/guides/sidebar.mdx

```diff
@@ -174,15 +172,13 @@ starlight({
 <SidebarPreview
 	config={[
 		{
-			label: 'Guides',
+			label: '星座',
 			items: [
-				{ label: 'Components', link: '/guides/components/' },
-				{ label: 'Internationalization (i18n)', link: '/guides/i18n/' },
+				{ label: '船底座', link: '' },
+				{ label: '半人马座', link: '' },
 				{
-					label: 'advanced',
-					items: [
-						{ label: 'Project Structure', link: '/guides/project-structure/' },
-					],
+					label: '周期',
```

Comment:
```suggestion
					label: 'seasonal',
```

In the case of nested folders in autogenerated groups, as we do not have an API for this, the folder name will be used (which is even more obvious now so I think it helps a bit)